cmd/juju/backups/remove.go

```diff
@@ -58,18 +74,75 @@ func (c *removeCommand) Run(ctx *cmd.Context) error {
 			return err
 		}
 	}
-	client, err := c.NewAPIClient()
+
+	client, apiVersion, err := c.NewGetAPI()
 	if err != nil {
 		return errors.Trace(err)
 	}
 	defer client.Close()
 
-	err = client.Remove(c.ID)
-	if err != nil {
-		return errors.Trace(err)
+	if apiVersion < 2 {
+		if c.KeepLatest {
```

Comment:
This doesn't need to be nested i.e. `if apiVersion < 2 && c.KeepLatest {`

Reply:
done